src/js/main.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 import { initializeTranslations } from './translations.js';
+import { addInputsChangeEvents } from './settings.js';
 
 document.addEventListener('DOMContentLoaded', () => {
 	initializeTranslations();
-
-	console.log('Hello World!');
+	addInputsChangeEvents();
```

Comment:
Ce ai facut aici in main, trebuie mutat in settings.js. Iar de aici din main poti sterge.
Ideea e sa separam codul js pentru settings in settings, iar in main doar ce e pentru main.

document.addEventListener('DOMContentLoaded', () => {
addInputsChangeEvents();
});

Reply:
Nu imi mergea inainte daca nu o puneam in main, dar am gasit de ce. Am uitat sa pun si settings.js in fisierul html.